python/ray/train/predictor.py

```diff
@@ -4,8 +4,8 @@
 import numpy as np
```

Comment:
Should we add this to the init for ray.train?

Reply:
I'm a bit wary of that, considering this is an abstract class.

Reply:
Hmm we have BaseTrainer in the top level right? And that too is an abstract class.

Reply:
Let's remove base trainer then